### ISSUE TITLE
Plugin does not work with custom backoffice URL

### ISSUE THREAD

**Issue by ondrejpialek** [skttl/umbraco-doc-type-grid-editor#138]
The issue is that the resources JS file contains URLs that have `umbraco` hardcoded in them. This does not work if the backoffice has been moved to a different URL.

The URLs could be constructed using `Umbraco.Sys.ServerVariables.umbracoSettings.umbracoPath` instead to support custom backoffice URLs.

**Comment by ondrejpialek** [comment]
As a workaround add this to the backoffice JS files:
```
angular.module('umbraco.services').config([
    '$httpProvider',
    function ($httpProvider) {
        $httpProvider.interceptors.push(function ($q) {
            return {
                'request': function (config) {
                    //some plugins have hardcoded backoffice urls :(
                    //this will fix it without needing to modify their source
                    config.url = config.url.replace(/^\/umbraco\//, '/hax0rproofbackoffice/');
                    return config;
                }
            };
        });
    }
]);
```

**Comment by skttl** [comment]
Fixed in 7323c39
